depinject/internal/codegen/file.go

```diff
@@ -61,7 +61,7 @@ func NewFileGen(file *ast.File, codegenPkgPath string) (*FileGen, error) {
 		if spec.Name != nil {
 			name := spec.Name.Name
 			if name == "." {
-				return nil, fmt.Errorf(". package imports are not allowed")
+				return nil, errors.New(". package imports are not allowed")
```

Comment:
Side note: we should have a unified way to do error returns in our code.  In some places in this PR its going `errors.New` -> `fmt.Errorf` and in other places its going the other way.

My opinion is that we should just always use `fmt.Errorf()` so that if necessary we can extend with more info or wrap errors (but dont have to)

Reply:
I'm fine with either way, but i don't necessarily agree with making a strict rule about going either way. At the end of the day, virtually all Go devs are familiar with both and feel like making this a rule would do little else than slow PR review cycles for something fairly arbitrary